Volume/Views/BrowserView.swift

```diff
@@ -80,6 +84,9 @@ struct BrowserView: View {
         .padding([.leading, .trailing], 16)
         .padding([.top, .bottom], 8)
         .background(Color.volume.backgroundGray)
+        .onAppear(perform: {
```

Comment:
This can be a trailing closure. Anyway, you can delete the State variable and replace it with a computer var. That way we remove an unnecessary reprender (once for the userData publisher, then again for the State var).

Reply:
Would you like me to put that computed var within the declaration of the view?  I didn't do this earlier because I didn't see it done anywhere in the codebase, but I can definitely change to that.

<img width="725" alt="Screen Shot 2021-02-17 at 3 20 13 PM" src="https://user-images.githubusercontent.com/56354603/108263000-aa568600-7133-11eb-94e7-e0b6c7d9f8b0.png">
This is what I mean.  Would you like me to put this somewhere different?